packages/desktop-client/src/components/ManageRules.js

```diff
@@ -588,20 +588,29 @@ export default function ManageRules({
   }, []);
 
   function onCreateRule() {
+    let rule = {
+      stage: null,
+      conditions: [
+        {
+          field: 'payee',
+          op: 'is',
+          value: payeeId || null,
+          type: 'id'
+        }
+      ],
+      actions: [
+        {
+          op: 'set',
+          field: 'category',
+          value: payeeId || null,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this correct? `payeeId` being set on the `category` field.

Reply:
Apologies, pre-coffee commit. Fixed